docs/data/data-grid/filtering/ServerFilterGrid.tsx

```diff
@@ -12,13 +12,13 @@ export default function ServerFilterGrid() {
     setQueryOptions({ filterModel: { ...filterModel } });
   }, []);
 
-  const { isLoading, data } = useQuery(queryOptions);
+  const { isLoading, rows } = useQuery(queryOptions);
 
   return (
     <div style={{ height: 400, width: '100%' }}>
       <DataGrid
-        rows={data}
-        columns={columns}
+        rows={rows}
+        {...data}
```

Comment:
```suggestion
        {...data}
        rows={rows}
```
I thaught it would be the opposite. Maybe it's the same

Reply:
I don't know if it's the same, but it's clearly a bad habit, fixed :+1: